specs/core/1_custody-game.md

```diff
@@ -272,6 +272,16 @@ def get_custody_chunk_count(crosslink: Crosslink) -> int:
     return crosslink_length * chunks_per_epoch
 ```
 
+### `get_bitfield_bit`
```

Comment:
We don't need this function, right?

Reply:
I did not change this logic at the moment because that will likely change quite a bit anyway. I think in the future, we will not need it, but I wanted to keep the old implementation alive for now.